notebooks/ullyses-walkthrough.ipynb

```diff
@@ -156,10 +187,9 @@
     "|--------------------------------------------------------------------------------------------------------------|-----------|---------------|-------------------------------------------------|-------------------|-------|\n",
```

Comment:
this sentence is a little hard to parse. suggest rephrasing: "The file names for ULLYSES science data products follow a naming scheme which encodes the target designation and the instruments and observing configuration(s) that contribute to the product. However, not all products will appear in the early releases. File names have the form" -> "The file names for ULLYSES science data products follow a specific naming scheme. This includes the target name, instrument(s), and observing configuration(s) that contributed to the product. However, not all products will appear in the early releases. File names have the form"

Reply:
Also missing a comma between the `<target>` and `<version>` comment